<?xml version="1.0"?>
<launch>
    <node name="table_tennis_agent" pkg="table_tennis" type="table_tennis_agent.py" respawn="false" output="screen"/>
</launch>